### PR TITLE
tables funky fresh

### PR DESCRIPTION
**Pull request by benauriemma**

### CODE REVIEW

File: src/components/elementView/elementTable.tsx [hunk 1]
```diff
@@ -101,10 +103,11 @@ interface ElementTableState {
 }
 
 interface ElementTableProps {
-  isDecommissioned: boolean;
+  isDecommissioned: boolean; // TODO: remove this and use assetType instead
```
Comment (benauriemma) [inline src/components/elementView/elementTable.tsx:106]: Probably not going to do this because it's correct as is, just a little redundant

Comment (benauriemma) [reply src/components/elementView/elementTable.tsx:106]: ev4 amiright

**Comment by benauriemma** [comment]
Places to mark tables stale:

Asset detail edit
- racked table*
- stored table*

Asset detail decommission
- racked table*
- stored table*
- decommissioned table*

Asset detail delete
- racked table*
- stored table*

Racked asset table edit
- racked table*
- stored table*

Racked asset table decommission
- racked table*
- decommissioned table*

Offline asset table edit
- racked table*
- stored table*

Offline asset table decommission
- offline table*
- decommissioned table*

Model detail edit
- racked table*
- stored table*
- model table*

Model detail delete
- model table*

Model table edit
- racked table*
- stored table*
- model table*

Model table delete
- model table*

Datacenter edit
- racked table*

Offline site edit
- stored table - THIS ONE ISN'T DONE BECAUSE I DON'T THINK IT'S IMPLEMENTED YET

User table delete
- racked table*
- stored table*

change plan detail view resolve conflict
- racked table*
- stored table*

change plan detail view remove change
- racked table*
- stored table*
- decommissioned table*

change plan execute
- racked table*
- stored table*
- decommissioned table*

File: src/store/reducers/state.tsx [hunk 2]
```diff
@@ -82,6 +91,36 @@ const setBrowser = (state: any, action: any) => {
...
 
+const markTablesStale = (state: any, action: any) => {
+  return Object.assign({}, state, {
+        rackedAssetDataIsStale: action.staleTables.includes(TableType.RACKED_ASSETS),
+        storedAssetDataIsStale: action.staleTables.includes(TableType.STORED_ASSETS),
+        decommissionedAssetDataIsStale: action.staleTables.includes(TableType.DECOMMISSIONED_ASSETS),
+        modelDataIsStale: action.staleTables.includes(TableType.MODELS),
+      });
+};
+const markTableFesh = (state: any, action: any) => {
```
Comment (clairecfu) [inline src/store/reducers/state.tsx:102]: is this supposed to say... markTableFresh?

File: src/store/reducers/state.tsx [hunk 3]
```diff
@@ -104,6 +143,10 @@ const reducer = (state = initialState, action: any) => {
       return setPermissionState(state, action);
     case actionTypes.SET_BROWSER_TYPE:
       return setBrowser(state, action);
+    case actionTypes.MARK_TABLES_STALE:
+      return markTablesStale(state, action);
+    case actionTypes.MARK_TABLE_FRESH:
+      return markTableFesh(state, action);
```
Comment (clairecfu) [inline src/store/reducers/state.tsx:149]: tables funky fesh

**Pull request merged**